index.html

```diff
@@ -8,7 +8,7 @@
     <script
       src="https://code.jquery.com/jquery-3.2.1.min.js"
       integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
-
+    <script src="app.js"></script>
```

Comment:
You never linked to your JavaScript file in your HTML file. This should've been one of the first things you caught, when none of the `console.log`s in your JavaScript file printed. That is the whole purpose of having `console.log`s in your `app.js` file.